docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -70,6 +73,24 @@ The data nodes that are part of a **scenario**, **pipeline** or **task** can be
     task.sales_history
     ```
 
+Data nodes can be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^` with their config_id.
+It returns the list of all existing data nodes instantiated from the config_id provided as a parameter.
```

Comment:
"It" -> "This method"
"config_id provided as a parameter" -> "indicated configuration identifier"